packages/@sanity/mutator/src/jsonpath/extract.ts

```diff
@@ -1,5 +1,13 @@
 import {extractAccessors} from './extractAccessors'
 
+/**
+ * Extracts a value for the given JsonPath
+ *
+ * @param path - Path to extract
+ * @param value - Value to extract from
+ * @returns An array of objects from the given path
+ * @internal
```

Comment:
Since the method is exported, it needs to be marked as either `public` or `beta` - this one has remained unchanged for a long time, so I think we're good with marking this as public.
```suggestion
 * @public
```